src/js/lib/Scrollspy.jsx

```diff
@@ -95,22 +98,21 @@ export class Scrollspy extends React.Component {
       return false
     }
     const rect = el.getBoundingClientRect()
-    const winH = window.innerHeight
-    const doc = document
-    const scrollTop = doc.documentElement.scrollTop || doc.body.scrollTop
-    const scrollBottom = scrollTop + winH
+    const root = this._rootNode
+    const rootRect = root.getBoundingClientRect()
+    const scrollTop = root.scrollTop
+    const scrollBottom = scrollTop + rootRect.height
     const elTop = rect.top + scrollTop + this.props.offset
     const elBottom = elTop + el.offsetHeight
 
     return (elTop < scrollBottom) && (elBottom > scrollTop)
   }
 
   _isAtBottom () {
-    const doc = document
-    const body = doc.body
-    const scrollTop = (doc.documentElement && doc.documentElement.scrollTop) || body.scrollTop
```

Comment:
What browser versions does this aim to support? From what I can tell `body.scrollTop` support is deprecated. Is there another use here that I am missing? If so we can add this to the `_rootNode` get function.